public/app/features/dashboard-scene/pages/DashboardScenePageStateManager.ts

```diff
@@ -61,14 +63,18 @@ export class DashboardScenePageStateManager extends StateManagerBase<DashboardSc
   // This is a simplistic, short-term cache for DashboardDTOs to avoid fetching the same dashboard multiple times across a short time span.
   private dashboardCache?: DashboardCacheEntry;
 
+  constructor(state: DashboardScenePageState) {
+    super(state);
+
+    appEvents.subscribe(ReloadDashboardEvent, ({ payload }) => this.reloadDashboard(payload));
+  }
+
   // To eventualy replace the fetchDashboard function from Dashboard redux state management.
   // For now it's a simplistic version to support Home and Normal dashboard routes.
-  public async fetchDashboard({
-    uid,
-    route,
-    urlFolderUid,
-    keepDashboardFromExploreInLocalStorage,
-  }: LoadDashboardOptions): Promise<DashboardDTO | null> {
+  public async fetchDashboard(
+    { uid, route, urlFolderUid, keepDashboardFromExploreInLocalStorage }: LoadDashboardOptions,
```

Comment:
there should be no keepDashboardFromExploreInLocalStorage anymore, can you pull from main?

Reply:
Pulled latest main.